samples/bluetooth/mesh/chat/prj.conf

```diff
@@ -55,7 +55,6 @@ CONFIG_BT_MESH_DK_PROV=y
 # Enable Bluetooth mesh models debug logs
 CONFIG_BT_DEBUG_LOG=y
 CONFIG_BT_MESH_DEBUG=y
-CONFIG_BT_MESH_DEBUG_MODEL=y
```

Comment:
What is the reason for disabling model debugging here but leaving mesh stack debugging? App demonstrates vendor model implementation.

Reply:
It is actually not needed. Non of models that uses `CONFIG_BT_MESH_LOG_LEVEL` is used in chat sample.

Reply:
Actually, config server uses it, but doubt it needs logging. Chat model does logging differently.